packages/jest-cli/src/cli/args.ts

```diff
@@ -377,8 +377,8 @@ export const options = {
   moduleNameMapper: {
     description:
       'A JSON string with a map from regular expressions to ' +
-      'module names that allow to stub out resources, like images or ' +
-      'styles with a single module',
+      'module names or to arrays of module names that allow to stub ' +
+      'out resources, like images or styles with a single module',
     type: 'string',
```

Comment:
Does this work as comma separated from the terminal?

Reply:
This argument accepts json string. I have updated tests as well and it accepts array fine https://github.com/facebook/jest/pull/9465/files#diff-9998297f84a95a62c8112e3c6075718c